src/main/kotlin/wu/seal/jsontokotlin/JsonSchemaDataClassGenerator.kt

```diff
@@ -0,0 +1,73 @@
+package wu.seal.jsontokotlin
+
+import com.squareup.kotlinpoet.*
+import wu.seal.jsontokotlin.bean.jsonschema.JsonObjectDef
+
+class JsonSchemaDataClassGenerator(private val jsonObjectDef: JsonObjectDef) {
+  val classes = mutableListOf<TypeSpec>()
+
+  fun generate(className: String?) {
+    val clazz = className
+        ?: jsonObjectDef.title
+        ?: throw IllegalArgumentException("className cannot be null when jsonObjectDef.title is null")
+
+    val requiredFields = jsonObjectDef.required ?: emptyArray()
+    val properties = if (ConfigManager.isOrderByAlphabetical) jsonObjectDef.properties.toSortedMap() else jsonObjectDef.properties
```

Comment:
The same as isComment off, `OrderByAlphabetial`　also implemented by interceptors , no need to do it here

Reply:
Just as it original sequence is OK